client/src/app/shared/components/FilterToolbar/SearchFilterControl.tsx

```diff
@@ -28,11 +28,11 @@ export const SearchFilterControl = <TItem, TFilterCategoryKey extends string>({
   // Keep internal copy of value until submitted by user
   const [inputValue, setInputValue] = React.useState(filterValue?.[0] || "");
   // Update it if it changes externally
-  React.useEffect(() => {
-    setInputValue(filterValue?.[0] || "");
-  }, [filterValue]);
+  React.useEffect(() => setInputValue(filterValue?.[0] || ""), [filterValue]);
```

Comment:
FYI, in general we shouldn't use single-expression inline arrow functions in a useEffect because the return value of the useEffect callback matters (if you return a function there it will be called as cleanup before re-running the effect). I don't think `setInputValue` returns anything so maybe this one is fine, but it's a convention (at least IMO) that I like to stay consistent about.